### PR TITLE
[v6r15] Retire hosts

### PR DESCRIPTION
**Pull request by Sbalbp**
Added new CLI command, retire <host>, that removes a host and all of its installations (if any) from the DB

### CODE REVIEW

File: FrameworkSystem/Client/SystemAdministratorClientCLI.py [hunk 1]
```diff
@@ -1199,6 +1199,47 @@ def do_showall( self, args ):
...
+          for installation in host[ 'Installations' ]:
+            result = client.removeInstallations( installation, {}, { 'HostName': option } )
+            if not result[ 'OK' ]:
+              self.__errMsg( result[ 'Message' ] )
+              break
+          # Finally remove the host
+          result = client.removeHosts( { 'HostName': option } )
+          if not result[ 'OK' ]:
+            self.__errMsg( result[ 'Message' ] )
+
```
Comment (fstagni) [inline FrameworkSystem/Client/SystemAdministratorClientCLI.py]: Maybe just add a message saying that host 'x' has been successfully removed.

File: FrameworkSystem/Client/SystemAdministratorClientCLI.py [hunk 2]
```diff
@@ -1199,6 +1199,47 @@ def do_showall( self, args ):
       if silentHosts:
         print "\n %d out of %d hosts did not respond" % ( len( silentHosts ), len( respondingHosts ) )
 
+  def do_retire( self, args ):
```
Comment (fstagni) [inline FrameworkSystem/Client/SystemAdministratorClientCLI.py]: What if we call it "remove" ?

**Comment by atsareg** [comment]
I do not think that remove is a good name for this command. I suggest that it is "uninstall host" like there is "uninstall db" command

**Comment by fstagni** [comment]
Review OK

**Pull request merged**